lms/djangoapps/discussion/templates/discussion/discussion_profile_page.html

```diff
@@ -45,34 +57,59 @@
 
 <%include file="../courseware/course_navigation.html" args="active_page='discussion'" />
 
-<section class="container">
+<%block name="content">
+<section class="discussion inline-discussion discussion-user-profile-board container" 
+         data-course-id="${course.id}"
```

Comment:
What's the distinction between props passed to the board as data attrs here, or as a config object to the factory on lines 35-46? I see some overlap - e.g. `course.id` is set in both places.